<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="2">

  <testsuite name="listing"
    errors="0" failures="1" hostname="" id="" package="listing" skipped="1"
    tests="2" time="0.000" timestamp="Wed, 11 Nov 2015 14:27:19 GMT">
  
    <testcase name="Listing Item Count" time="4.475" assertions="1">  <failure message="Checking that link is a uoguelph website.">    at Object.module.exports.Listing Item Count (/home/autobox/Desktop/Unit Tests/Unit-Tests/listing.js:13:15)
    at Module.call (/home/autobox/node_modules/nightwatch/lib/runner/module.js:63:34)
    at /home/autobox/node_modules/nightwatch/lib/runner/testcase.js:97:29
    at _fulfilled (/home/autobox/node_modules/nightwatch/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/home/autobox/node_modules/nightwatch/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/home/autobox/node_modules/nightwatch/node_modules/q/q.js:796:13)
    at /home/autobox/node_modules/nightwatch/node_modules/q/q.js:556:49
    at runSingle (/home/autobox/node_modules/nightwatch/node_modules/q/q.js:137:13)
    at flush (/home/autobox/node_modules/nightwatch/node_modules/q/q.js:125:13)
    at doNTCallback0 (node.js:419:9)</failure>

    
    </testcase>
  
  
    
    <testcase
      name="Listing Count">
      <skipped />
    </testcase>
    
  
  </testsuite>
</testsuites>
